billTable.js

```diff
@@ -0,0 +1,150 @@
+let currTable, itemCount, targetId, orderData;
+let tableHeader=document.querySelector(".table-name");
+let popUp=document.querySelector(".table-list");
+let totalTables=document.querySelectorAll(".table-item");
+let billAmount=document.getElementById("total");
+let tableBody = document.querySelector(".bill");
```

Comment:
can you use const over let if the value is not changing?
can you also replace everywhere in the code ?

Reply:
I made the respective changes .